Segment-ComScore/Classes/SEGComScoreIntegration.h

```diff
@@ -8,16 +8,13 @@
 
 #import <Foundation/Foundation.h>
 #import <Analytics/SEGIntegration.h>
-#import <ComScore/CSComScore.h>
+#import <ComScore/ComScore.h>
 
 
 @interface SEGComScoreIntegration : NSObject <SEGIntegration>
 
 @property (nonatomic, strong) NSDictionary *settings;
-@property (nonatomic, strong) Class comScoreClass;
```

Comment:
This was written this way to be able to write tests. I guess we didn't for v1, but we should for v2. (examples https://github.com/segment-integrations/analytics-ios-integration-mixpanel/blob/master/Example/Tests/Tests.m)